components/02-elements/dropdown-list/_dropdown-list.scss

```diff
@@ -78,156 +67,129 @@ $dropwodn-list-inner-item-color: $color-primary !default;
         display: block;
         width: 100%;
         padding: 0;
-        font-weight: 700;
-        font-size: $dropdown-list-font-size;
-        text-transform: uppercase;
+        font-size: $dropdown-list__font-size;
         cursor: pointer;
-        transition: box-shadow 0.5s;
-
-        @include mq($min-screen: $screen-m, $max-screen: $screen-xl) {
-            flex-basis: 24%;
-
-            &:hover {
-                box-shadow: 0 3px 18px 0 rgba(0, 0, 0, 0.13);
-            }
-        }
 
+        &--open,
         &.open {
-            @include mq($min-screen: $screen-m, $max-screen: $screen-xl) {
-                box-shadow: 0 3px 18px 0 rgba(0, 0, 0, 0.13);
+            & > .dropdown-list__label {
+                color: $dropdown-list__item-color-active;
+                background-color: $dropdown-list__item-bg-color-active;
             }
-
-            .filter__list {
-                margin-top: 10px;
+            .dropdown-list__icon:not(.dropdown-list__icon--inner) {
```

Comment:
Will be better to use `& > selector` instead of using `:not`

Reply:
corrected